### PR TITLE
fix(dependencies): add missing dependencies required for planner

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ module.exports = function (options) {
           'ENV': JSON.stringify(METADATA.ENV),
           'NODE_ENV': JSON.stringify(METADATA.ENV),
           'HMR': METADATA.HMR,
+          'API_URL': JSON.stringify(METADATA.FABRIC8_WIT_API_URL),
```
Comment (joshuawilson) [inline config/webpack.dev.js]: why is this needed?

Comment (dgutride) [inline config/webpack.dev.js]: The planner is still using that value internally @joshuawilson so we had to have it available

Comment (joshuawilson) [inline config/webpack.dev.js]: ok, for now but we need to fix it in Planner next.
